datadog_checks_dev/CHANGELOG.md

```diff
@@ -1,5 +1,46 @@
+* [Added] Annotate metadata validation. See [#10128](https://github.com/DataDog/integrations-core/pull/10128).
+* [Added] Annotate package validation. See [#10115](https://github.com/DataDog/integrations-core/pull/10115).
+* [Added] Annotate licenses. See [#10114](https://github.com/DataDog/integrations-core/pull/10114).
+* [Added] Annotate readme validations. See [#10116](https://github.com/DataDog/integrations-core/pull/10116).
+* [Added] Allow exclusion of specific branch for changelog generation. See [#10106](https://github.com/DataDog/integrations-core/pull/10106).
+* [Added] Annotate jmx metric validation. See [#10113](https://github.com/DataDog/integrations-core/pull/10113).
+* [Added] Annotate EULA and agent requirements validation. See [#10108](https://github.com/DataDog/integrations-core/pull/10108).
+* [Added] Annotate codeowners. See [#10107](https://github.com/DataDog/integrations-core/pull/10107).
+* [Added] Echo warning for unnecessary params used. See [#10053](https://github.com/DataDog/integrations-core/pull/10053).
+* [Added] Add borrower and pysmi logs to MIB compiler. See [#10074](https://github.com/DataDog/integrations-core/pull/10074).
+* [Added] Allow the use of ddtrace for E2E tests. See [#10082](https://github.com/DataDog/integrations-core/pull/10082).
+* [Added] Disable generic tags. See [#10027](https://github.com/DataDog/integrations-core/pull/10027).
+* [Added] Add support for manifest v2 validations. See [#9968](https://github.com/DataDog/integrations-core/pull/9968).
+* [Added] Add critical service check test to integration template. See [#10063](https://github.com/DataDog/integrations-core/pull/10063).
+* [Added] Add support for testing new versions of products. See [#9945](https://github.com/DataDog/integrations-core/pull/9945).
+* [Added] Update release tooling to support `datadog_checks_dependency_provider`. See [#10046](https://github.com/DataDog/integrations-core/pull/10046).
+* [Added] Add pytest plugin dependency to handle flakes. See [#10043](https://github.com/DataDog/integrations-core/pull/10043).
+* [Added] Annotate dashboard and recommended monitors validation. See [#9899](https://github.com/DataDog/integrations-core/pull/9899).
+* [Added] Annotating display_queue. See [#9944](https://github.com/DataDog/integrations-core/pull/9944).
+* [Fixed] Add Avi Vantage to INTEGRATION_LOGS_NOT_POSSIBLE . See [#9667](https://github.com/DataDog/integrations-core/pull/9667).
+* [Fixed] Remove annotation for unnecessary warning. See [#10124](https://github.com/DataDog/integrations-core/pull/10124).
+* [Fixed] Fix mypy tests. See [#10134](https://github.com/DataDog/integrations-core/pull/10134).
+* [Fixed] Bump mypy. See [#10119](https://github.com/DataDog/integrations-core/pull/10119).
+* [Fixed] Use regex to parse for http wrapper instead of reading by line. See [#10055](https://github.com/DataDog/integrations-core/pull/10055).
```

Comment:
```suggestion
* [Fixed] Use Regex to parse for HTTP wrapper instead of reading by line. See [#10055](https://github.com/DataDog/integrations-core/pull/10055).
```